site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -178,102 +148,156 @@ files. To see if this is the problem, follow these steps:
     external extension was not installed, repeat these steps until you do not see an error in the
     Console application.
 
+{% endDetails %}
+
+### Linux {: #preference-linux }
+
+1. Create a JSON file with the name of the extension ID. For example: `aaabbbcccdddeeefff.json`.
+1. Place it in one of the folders listed below:
+    - `/opt/google/chrome/extensions/`
+    - `/usr/share/google-chrome/extensions/`
+
+1. The following list describes extension installation from the Chrome Web Store, a CRX file or a
+   personal server:
+
+    - To install a **Chrome Web Store** extension, specify the update URL with the field name
+      "external_update_url". For example:
+      ```json
+      {
+        "external_update_url": "https://clients2.google.com/service/update2/crx"
+      }
+      ```
+    - To install the extension from a **CRX file**, specify the location in "external_crx" and the
+      version in "external_version". For example:
+      ```json
+      {
+        "external_crx": "/home/share/extension.crx",
+        "external_version": "1.0"
+      }
+      ```
+    - To install the extension **[hosted on a personal server][xml-update-url]**, the
+      "external_update_url" field has to point to the xml file, like in the following example:
+      ```json
+      {
+        "external_update_url": "http://myhost.com/mytestextension/updates.xml"
+      }
+      ```
+1.  Save the JSON file.
+1.  Launch Google Chrome and go to **chrome://extensions**; you should see the extension listed.
+
+{% Aside %}
+
+Use `chmod` if necessary to make sure that the `aaabbbcccdddeeefff.json` files are world-readable.
+Check the [preference file common mistakes FAQ][section-faq-preference] for additional help.
+
+{% endAside %}
+
+### Supported Locales {: #preference-locales }
+
+If you would like to install extension only for some browser locales, you can list supported locales
+in field name "supported_locales". Locale may specify parent locale like "en", in this case the
+extension will be installed for all English locales like "en-US", "en-GB", etc. If another browser
+locale is selected that is not supported by the extension, the external extensions will be
+uninstalled. If "supported_locales" list is missing, the extension will be installed for any locale.
+For example:
+
+```json
+{
+  "external_update_url": "https://clients2.google.com/service/update2/crx",
+  "supported_locales": [ "en", "fr", "de" ]
+}
+```
+
```

Comment:
Looking at the code [1], it does not appear this is supported in the Windows registry path.

[1] https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/extensions/external_registry_loader_win.cc;l=1;drc=643cdf61903e99f27c3d80daee67e217e9d280e0

Reply:
Thanks for the clarification. For additional context, I was thinking that we may want to make this section top level rather than a child of the "Using a preferences file" section.

Reply:
I'm leaving it as it is, since only the preference file supports this